source/write/insert.txt

```diff
@@ -0,0 +1,221 @@
+
+.. note:: Duplicate _id Errors
+
+   Setting duplicate ``_id`` values in a collection violates unique
+   index constraints, which causes the driver to return a ``WriteError`` from
+   the ``insertOne()`` method or a ``BulkWriteError`` from the
+   ``insertMany()`` method.
+
+To learn more about the ``_id`` field, see the
+:manual:`Unique Indexes </core/index-unique/>` guide in the
+{+mdb-server+} manual.
+
+To learn more about document structure and rules, see the
+:manual:`Documents </core/document>` guide in the {+mdb-server+} manual.
+
+Insert One Document
+-------------------
+
+To add a single document to a MongoDB collection, call the ``insertOne()``
+method and pass the document you want to add.
+
+The following example inserts a document into the ``restaurants``
+collection:
+
+.. literalinclude:: /includes/write/insert.kt
+   :start-after: start-insert-one
+   :end-before: end-insert-one
+   :language: kotlin
+   :copyable:
+   :dedent:
+
+Insert Multiple Documents
+-------------------------
+
+To add multiple documents to a MongoDB collection, user the ``insertMany()``
+method and pass a list of documents you want to add.
+
+The following example inserts a list of documents into the
+``restaurants`` collection:
+
+.. literalinclude:: /includes/write/insert.kt
+   :start-after: start-insert-many
+   :end-before: end-insert-many
+   :language: kotlin
+   :copyable:
+   :dedent:
+
+Modify Insert Behavior
+----------------------
+
+The ``insertOne()`` method optionally accepts a parameter
+that sets options to configure the insert operation.
+If you don't specify any options, the driver performs the insert
+operation with default settings.
+
+The following table describes the setter methods that you can use to
+configure an ``InsertOneOptions`` instance:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Property
+     - Description
+
+   * - ``bypassDocumentValidation()``
+     - | If set to ``true``, allows the driver to ignore
+         :manual:`document-level validation </core/schema-validation>`. 
+       | Defaults to ``false``.
+
+   * - ``comment()``
+     - | Sets a comment to attach to the operation. For more information, see the :manual:`insert command
+         fields </reference/command/insert/#command-fields>` guide in the
+         {+mdb-server+} manual for more information.
+
+You can set the preceding settings on the ``insertMany()`` method
+by configuring an ``InsertManyOptions`` instance. You can also use the
+``ordered()`` method setter method to specify the order in which the driver
+inserts documents into MongoDB. If set to ``true``, the driver sends documents to the
+server in the order provided. If an error occurs, the driver and server
+cancel all remaining insert operations. The driver performs ordered
+inserts by default.
+
+Modify Insert Example
+~~~~~~~~~~~~~~~~~~~~~
+
+The following code creates options and uses the ``bypassDocumentValidation()`` method to
```

Comment:
```suggestion
The following code sets the ``bypassDocumentValidation()`` option to
```

Reply:
It might be confusing to use option and method interchangeably, I will workshop different phrasing.